### ISSUE TITLE
Add sticker counts in the main pack view

### ISSUE THREAD

**Issue by noblereign** [Liusef/TelegramStickerManager#75]
It would be nice to be able to see how many stickers are in a pack. It'd be much more convenient to tell if I have room for more stickers at a glance. Also, the Telegram clients I use don't really make this information easy to see (on Android you have to make a link to the pack and tap on it, with Unigram and Telegram Desktop I'm pretty sure it just isn't anywhere at all) so having one easy spot to view it would be great too.

The sticker count should probably be within the main pack view. The home screen, while a tempting choice, may be a bit overwhelming if you have a lot of packs.
Yet again, another terrible mockup for your viewing pleasure:
![image](https://user-images.githubusercontent.com/2935979/161152226-33e3f806-cbc3-4d9b-a3fe-38b8265554e4.png)

**Comment by Liusef** [comment]
I'll see if i can add it to the pack page

Just if you were curious, adding to the home screen wouldn't be possible. The application only caches the sticker image locally, and every time the pack page is open it pulls the metadata for the pack and each sticker from tdlib (idk how tdlib caches internally). This was to speed up loading times, reduce data usage, and make managing the cache for the program a little more manageable.

Basically, the app has no idea what stickers (let alone how many stickers) are in each pack until you actually click on it.
